src/app/home/getServices.js

```diff
@@ -242,14 +250,15 @@ const getServices = async (req, res) => {
 
   const userPireanServices = services.filter((value) => pireanServices.includes(value.name));
   ///testing here for new services use sortby accessGranted to get the latest one
-  let nowDate = new Date().getDate();
+  let nowDate = new Date().getDate(); //used to determine last logged 24 hrs
   const checklastestaddition = services.filter((x) => {
     return (
-      new Date(x.accessGrantedOn).getDate() >= new Date(last_login).getDate() &&
+      new Date(x.accessGrantedOn).getDate() >= new Date(user.claims.last_login).getDate() &&
       new Date(x.accessGrantedOn).getDate() <= nowDate
     );
   });
   if (checklastestaddition.length > 0 && (res.locals.flash === undefined || res.locals.flash.title === undefined)) {
+    ///does this exist in userbanner?
```

Comment:
do we need the comment line? if yes please explain in more detail

Reply:
its a place holder for the next job I take whoich is to do with displaying the banner for new services for only twentyfour hours and persistent close option for user